digits/views.py

```diff
@@ -19,6 +19,11 @@
 from digits.utils import errors
 from digits.utils.routing import request_wants_json
 from digits.log import logger
+from flask import Flask, request, render_template
+from workspaces import get_workspace
+import shutil
+import os
+from boto.ec2.autoscale import scheduled
```

Comment:
What is this for? If you're going to add a new library, you have to add it to the list of requirements. But it doesn't look like you're actually using this import ...